app/Repositories/Metric/AbstractMetricRepository.php

```diff
@@ -58,6 +58,11 @@ protected function getTableName()
         $connection = $this->config->get('database.connections.'.$driver);
         $prefix = $connection['prefix'];
 
-        return $prefix.'metrics';
+        $return = (object) array(
```

Comment:
Infact, it's better to just make two methods here than returning something like this.